cli/args/flags.rs

```diff
@@ -2875,7 +2877,7 @@ Start a server defined in server.ts, watching for changes and running on port 50
         .long("host")
         .help("The TCP address to serve on, defaulting to 0.0.0.0 (all interfaces)")
         .value_parser(serve_host_validator),
-    )
+    ).arg(Arg::new("open_site").long("open").help("Open the browser on the address that the server is running on.").action(ArgAction::SetTrue))
```

Comment:
Did you run `tools/format.js` before pushing? This seems not formatted

Reply:
yes I did, and did it again now, still the same.